src/server_manager/web_app/data_formatting.ts

```diff
@@ -49,6 +49,9 @@ function getDataFormattingParams(numBytes: number): FormatParams {
 function makeDataAmountFormatter(language: string, params: FormatParams) {
   // We need to cast through `unknown` since `tsc` mistakenly omits the 'unit' field in
   // `NumberFormatOptions`.
+  if (!language) {
```

Comment:
Let's revert this. Language is required and it should be the caller's responsibility to enforce that.

Reply:
Reverted